content/water-data-science-2021.md

```diff
@@ -0,0 +1,94 @@
+
+
+Advancing machine learning architectures for water prediction {#pgdl}
+--------------------
+
+<div class="grid-row">
+{{< figure src="/static/FY21DataSci/pgdl_flubber_small.gif" alt="Process-guided deep learning predictions of stream temperature" class="side-by-side" >}}
+
+<p class="side-by-side" >
+There is swirling mix of excitement, fear, misunderstanding, and raw wonder concerning the role of machine learning in water prediction. In the Data Science branch, we’re excited about the potential use of ML in all kinds of USGS projects. But being part of the USGS, we also understand the positive modeling legacy our agency has, which could be tarnished by misuse or blind adoption of ML. A lot of the fear surrounding ML in environmental sciences comes from a combination of how well it seems to perform in prediction and how effortless applying it to the problem de jour seems to be. There is a lot of nuance to each of these assumptions, as we rarely have enough data to build a truly generalizable environmental ML model, and while these models may be faster to spin up, major challenges exist for dealing with data sparsity, estimating uncertainty, and applying knowledge-based constraints that require custom architectures and cutting-edge research. We have partnerships with the Kumar lab at U-MN, the Jia lab at Pitt, and the Shen lab at Penn State to collaborate on the exciting field of "<a href="https://labs.waterdata.usgs.gov/visualizations/gages-through-the-ages/index.html#/">Knowledge Guided Machine Learning</a>". 
+</p>
+</div>
+We’re focusing heavily in 2021 on a subset of KGML - which we’re calling Process-Guided Deep Learning - to make improved predictions of water quality or quantity that are guided by physical models and laws (such as conservation of energy, see <a href="https://agupubs.onlinelibrary.wiley.com/doi/full/10.1029/2019WR024922">here</a>). These are advanced hybrid models that take the best of what our field has to offer for physics-based models and the best machine learning frameworks for spatiotemporally-aware models. The results of these collaborations have been exciting, with large improvements in our predictive accuracy and ability to transfer these models into unseen conditions and still make good predictions. We are currently working on the prediction of water temperature in lakes and streams with these methods, and also working on similar concepts for predicting stream discharge. The place for machine learning in USGS will continue to grow in the next several years and KGML and PGDL will generate some very interesting conversations regarding environmental modeling architectures. 
+
+
+Forecasting changes to water resources {#data-assimilation}
+--------------------
+
+<div class="grid-row">
+{{< figure src="/static/FY21DataSci/sequential_DA.png" alt="Sequential data assimilation forecasting" class="side-by-side" >}}
```

Comment:
Same here, the alt text should be descriptive.  There should also be a caption for the image (using `caption=`) which can be markdown.